src/Umbraco.Cms.Integrations.Automation.Zapier/Components/NewContentPublishedNotification.cs

```diff
@@ -30,24 +29,27 @@ public NewContentPublishedNotification(ZapConfigService zapConfigService, Zapier
 
         public void Handle(ContentPublishedNotification notification)
         {
+            var triggerHelper = new TriggerHelper(_zapierService);
+
             foreach (var node in notification.PublishedEntities)
             {
-                var zapContentConfig = _zapConfigService.GetByName(node.ContentType.Name);
-                if (zapContentConfig == null || !zapContentConfig.IsEnabled) continue;
-
-                var content = new Dictionary<string, string>
+                if (_zapierSubscriptionHookService.TryGetByAlias(node.ContentType.Alias, out var zapContentConfigList))
                 {
-                    { Constants.Content.Id, node.Id.ToString() },
-                    { Constants.Content.Name, node.Name },
-                    { Constants.Content.PublishDate, DateTime.UtcNow.ToString() }
-                };
-
-                var t = Task.Run(async () => await _zapierService.TriggerAsync(zapContentConfig.WebHookUrl, content));
-
-                var result = t.Result;
-
-                if (!string.IsNullOrEmpty(result))
-                    _logger.LogError(result);
+                    var content = new Dictionary<string, string>
+                    {
+                        {Constants.Content.Id, node.Id.ToString() },
+                        {Constants.Content.Name, node.Name },
+                        {Constants.Content.PublishDate, DateTime.UtcNow.ToString("s") }
```

Comment:
This still looks to be sending only the ID, name and publish date of the content, but I was expecting now we would be sending all the field values of the content too.

Reply:
Fixed.